src/openrct2/paint/track/coaster/VirginiaReel.cpp

```diff
@@ -221,25 +219,25 @@ static void PaintVirginiaReelTrack25DegUp(
     {
         case 0:
             WoodenASupportsPaintSetupRotated(
-                session, kSupportType, WoodenSupportSubType::NeSw, direction, height, session.SupportColours,
+                session, supportType.wooden, WoodenSupportSubType::NeSw, direction, height, session.SupportColours,
```

Comment:
Same question here; passing `supportType.wooden` to a function that's already called `WoodenASupportsPaintSetup` makes it sound like it ought to be implied.